dev-resources/vectors/README.md

```diff
@@ -1 +1,14 @@
-These vectors were taken from Kalium's test suite.
+The vectors in the following folders were taken from Kalium's test suite:
```

Comment:
Unfortunately that is not quite right. Comments inline.

Reply:
Actually on second thought we should put READMEs in the individual source directories.

Please:
 1. add a README to `kdf` specifically mentioning where you got this
 2. delete this README

I'll add READMEs to the other directories.

Reply:
Done.